packages/module/src/ChatbotConversationHistoryNav/ChatbotConversationHistoryNav.tsx

```diff
@@ -268,6 +264,28 @@ export const ChatbotConversationHistoryNav: FunctionComponent<ChatbotConversatio
             )}
           </DrawerActions>
         </DrawerHead>
+        <div className="pf-chatbot__input">
+          <Title headingLevel="h3">
+            <Icon size="lg">
+              <OutlinedClockIcon />
+            </Icon>
+            {'  '}
```

Comment:
Can we remove this whitespace? Instead of using whitespace, we can space these out with CSS if needed. Let me know if you need help with this.